src/analysis/transcendental/liouville_constant.lean

```diff
@@ -0,0 +1,165 @@
+/-
+Copyright (c) 2020 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang
+-/
+
+import analysis.transcendental.liouville
+import data.nat.factorial
+
+/-!
+# Liouville constants
+
+This files contains a construction of a family of Liouville number.
+The most important property is that they are examples of transcendental real numbers.
+This fact is recorded in `is_liouville.is_transcendental_of_α`.
+-/
+
+namespace is_liouville
+
+noncomputable theory
+open_locale nat big_operators
+open set real finset
+
+variable (m : ℕ)
+
+local notation `α` := ∑' (i : ℕ), 1 / (m : ℝ) ^ i!
```

Comment:
Perhaps this should be defined as a function `def liouville_constant (m : \N) := `
and then we can still give it special local notation. The same applies to the next two definitions.

Reply:
The reason I choose notation is that I don't want to constantly unfold all the definitions. Is there any workaround?